File: lingting-grpc/src/main/java/live/lingting/framework/grpc/interceptor/AbstractServerInterceptor.java
package live.lingting.framework.grpc.interceptor;

import io.grpc.ServerInterceptor;
import live.lingting.framework.grpc.GrpcServer;

/**
 * @author lingting 2024-01-31 10:12
 */
public abstract class AbstractServerInterceptor implements ServerInterceptor {

	protected GrpcServer server;

	public void setServer(GrpcServer server) {this.server = server;}
}

File: lingting-grpc/src/main/kotlin/live/lingting/framework/grpc/interceptor/AbstractServerInterceptor.kt
package live.lingting.framework.grpc.interceptor

import io.grpc.ServerInterceptor
import live.lingting.framework.grpc.GrpcServer

/**
 * @author lingting 2024-01-31 10:12
 */
abstract class AbstractServerInterceptor : ServerInterceptor {

    var server: GrpcServer? = null

}
